tests/flags.cpp

```diff
@@ -0,0 +1,50 @@
+#include "openxr/openxr.hpp"
+#include <type_traits>
+
+#include <gtest/gtest.h>
+
+class OpenXrFlagsTest : public ::testing::Test {
+   protected:
+    void SetUp() override {}
+
+    void TearDown() override {}
+};
+
+TEST_F(OpenXrFlagsTest, flagsTest) {
+    {
+        xr::SpaceVelocityFlags flags = xr::SpaceVelocityFlagBits::LinearValid | xr::SpaceVelocityFlagBits::AngularValid;
+        EXPECT_EQ(flags & xr::SpaceVelocityFlagBits::LinearValid, xr::SpaceVelocityFlagBits::LinearValid);
+        EXPECT_EQ((flags & xr::SpaceVelocityFlagBits::LinearValid).operator bool(), true);
+        EXPECT_EQ(flags & xr::SpaceVelocityFlagBits::AngularValid, xr::SpaceVelocityFlagBits::AngularValid);
+        EXPECT_EQ((flags & xr::SpaceVelocityFlagBits::AngularValid).operator bool(), true);
+		XrSpaceVelocityFlags cflags = flags.operator size_t();
```

Comment:
is it really an `operator size_t`? I'd expect otherwise since that varies in size between platforms...